tests/js/jest.config.js

```diff
@@ -40,7 +41,10 @@ module.exports = {
 		'<rootDir>/build',
 	],
 	modulePathIgnorePatterns: [ '<rootDir>/.vscode' ],
-	transformIgnorePatterns: [ '<rootDir>/node_modules/(?!@material/web)/.*' ],
+	transformIgnorePatterns: [
+		'/node_modules/(?!(@material\\/web))/',
```

Comment:
This one should remain as it was before:
```suggestion
		'<rootDir>/node_modules/(?!@material/web)/.*',
```